### PR TITLE
Documentation: Add spork message / details to protocol-documentation.md

### PR DESCRIPTION
**Pull request by thephez**
Also:
 - Add place holders for implemented messages that are not documented yet
 - Fix a couple minor typos

### CODE REVIEW

File: dash-docs/protocol-documentation.md [hunk 1]
```diff
@@ -215,3 +215,70 @@ Masternodes use governance voting in response to new proposals, contracts, setti
...
+
+### TXLOCKREQUEST - "ix"
+
+Tx Lock Request
+
+### MNGOVERNANCESYNC - "govsync"
+
+Governance Sync
+
+### dseg
```
Comment (UdjinM6) [inline dash-docs/protocol-documentation.md]: DSEG - "dseg"

File: dash-docs/protocol-documentation.md [hunk 2]
```diff
@@ -215,3 +215,70 @@ Masternodes use governance voting in response to new proposals, contracts, setti
...
+
+???
```
Comment (UdjinM6) [inline dash-docs/protocol-documentation.md]: Masternode List/Entry Sync

Get Masternode list or specific entry

Comment (thephez) [inline dash-docs/protocol-documentation.md]: Got it.  Thanks!  I wasn't sure what to call this one.

**Comment by thephez** [comment]
You're welcome.  I needed some of this to work on my Wireshark plugin and thought it would be useful.  Your suggestions have been incorporated.

**Pull request merged**